tools/stress-cluster/cluster/kubernetes/stress-test-addons/deploy.ps1

```diff
@@ -0,0 +1,28 @@
+$ErrorActionPreference = 'Stop'
+Set-PSDebug -trace 1
+$subscriptionId = "2cd617ea-1866-46b1-90e3-fffb087ebf9b"
+$env:AZURE_STORAGE_ACCOUNT="stresstestcharts"
+$env:AZURE_STORAGE_KEY=$(az storage account keys list --subscription $subscriptionId --account-name $env:AZURE_STORAGE_ACCOUNT -o tsv --query '[0].value')
+if($LASTEXITCODE) {exit $LASTEXITCODE}
```

Comment:
I'd still copy the `Run` method from `provision.ps1` to avoid all this verbosity.